proposal.rst

```diff
@@ -168,27 +210,83 @@ Project Details
 
 **Describe any existing work and concepts on which your project is based.**
 
-FILLME
+My work will be based on the GSoC 2017 Project `Clickstream Mining for 
+Learning to Rank training data <https://trac.xapian.org/wiki/GSoC2017/LetorClickstream>`_. 
```

Comment:
Minor point, but since you are also proposing implementing further algorithms from the original paper and it sounds like some from other papers, it'd be good reference them here.

Reply:
Okay, I'll put the reference here